pkg/controllermanager/controller/project/project_activity_control.go

```diff
@@ -73,12 +73,18 @@ func (r *projectActivityReconciler) Reconcile(ctx context.Context, request recon
 	return reconcile.Result{}, nil
 }
 
-func (c *Controller) updateBackupEntryActivity(ctx context.Context, obj interface{}) {
+func (c *Controller) projectActivityBackupEntryAdd(ctx context.Context, obj interface{}) {
 	backupEntry, ok := obj.(*gardencorev1beta1.BackupEntry)
```

Comment:
you could re-use the same method for a couple of objects using ObjectMeta for checking the CreationTimestamp
```Go
func (c *Controller) projectActivityObjectAdd(ctx context.Context, obj interface{}) {
	objMeta, ok := obj.(*metav1.ObjectMeta)
	if !ok {
		return
	}

	// If the creationTimestamp of the object is less than 1 hour (experimental) from current time,
	// skip it. This is to prevent unnecessary reconciliations in case of GCM restart.
	if objMeta.GetCreationTimestamp().Add(time.Hour).UTC().Before(c.clock.Now().UTC()) {
		return
	}

	key := c.getProjectKey(ctx, objMeta.GetNamespace())
	if key == "" {
		return
	}

	c.projectActivityQueue.Add(key)
}
```

Reply:
we would need to get the objectMeta of the object first, to assign it to type (*metav1.ObjectMeta), right?

Reply:
Oh sorry, I was wrong. You can't get ObjectMeta directly. You can use `"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"` for it `obj.(*unstructured.Unstructured)`. 
This should include ObjectMeta and independent from the concrete type.

Reply:
I think using `meta.Accessor(obj)` is the better approach (and already used elsewhere in the code).